checkstyle-tester/launch.groovy

```diff
@@ -254,45 +253,13 @@ def postProcessCheckstyleReport(targetDir, repoName, repoPath) {
     Files.createLink(Paths.get(getOsSpecificPath("$siteDir", "index.html")),
         Paths.get(getOsSpecificPath("$siteDir", "checkstyle.html")))
```

Comment:
What about these items surrounding index/checkstyle HTML?
If HTML Java files are no longer supported, should we care if we have indexx/checkstyle HTML?

Reply:
Fixed.